ctagsdotd/vbnet.ctags

```diff
@@ -0,0 +1,12 @@
+# See additional-language.ctags for maintaining this file
```

Comment:
Suggestion: Add a link to https://learn.microsoft.com/en-us/dotnet/visual-basic/reference/language-specification/type-members in the comment so that is easier for someone to cross-check this.